bridges/docs/dockerhub-bridges-common-relay.README.md

```diff
@@ -0,0 +1 @@
+# bridges-common-relay
```

Comment:
@serban300 
I guess, we should exclude this dockerhub-*.README.md from verify-pallets-build.sh,
please, just update verify-pallets-build.sh in bridges repo, and next time it will come here

Reply:
Good point. Will do.